R/github.R

```diff
@@ -155,10 +155,11 @@ use_github <- function(organisation = NULL,
 
   ui_done("Setting remote {ui_value('origin')} to {ui_value(origin_url)}")
   use_git_remote("origin", origin_url)
+  git_push_first(default_branch, "origin")
```

Comment:
This seems like a more logical location to me: you create the remote origin and then push. But maybe there was good reason to do it later?

Reply:
Yeah I think I wanted the attempt to `use_github_links()` to happen _first_ so that's in DESCRIPTION before we push.